flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/checkpoints/CheckpointStatistics.java

```diff
@@ -154,6 +161,7 @@ private CheckpointStatistics(
 		this.persistedData = persistedData;
 		this.numSubtasks = numSubtasks;
 		this.numAckSubtasks = numAckSubtasks;
+		this.checkpointType = checkpointType;
```

Comment:
I forgot to mention in the other review: It's always good to call `Preconditions.checkNotNull` on nullable parameters. See other `status` as an example.

Reply:
sure I can do that.